src/toolchains.rs

```diff
@@ -334,6 +335,31 @@ impl Toolchain {
                 }
                 cmd
             }
+            (Some(script), Some(timeout)) => {
+                let mut cmd = Command::new("timeout");
+                let mut args: Vec<std::ffi::OsString> =
+                    vec![script.into(), format!("{}", timeout).into()];
```

Comment:
The order is reversed.

```suggestion
                    vec![format!("{}", timeout).into(), script.into()];
```